progs/symmetrc_mimc.py

```diff
@@ -0,0 +1,69 @@
+from random import randint
+from asyncio import gather
+from honeybadgermpc.field import GF
+from honeybadgermpc.mpc import PreProcessedElements, Subgroup
+from honeybadgermpc.elliptic_curve import Point, Jubjub
+from progs.jubjub import share_mul
+from progs.mimc import mimc_mpc, mimc_plain
+
+pp_elements = PreProcessedElements()
+field = GF(Subgroup.BLS12_381)
+CURVE = Jubjub()
+KEY_LENGTH = 32
+# GP: The generator of Jubjub curve, hardcode here
+GP = Point(5,
+           6846412461894745224441235558443359243034138132682534265960483512729196124138,
+           CURVE)
+
+
+async def key_generation(context):
+    """
```

Comment:
Great documentation here

Reply:
No sarcasm in code review please :o

Reply:
the documentation was actually great when I made that comment :(

Reply:
There was neat documentation here but was overwritten by my messed rebase -- apologies for the misunderstanding I brought lol